xray/plot/plot.py

```diff
@@ -452,6 +452,8 @@ def newplotfunc(darray, ax=None, xincrease=None, yincrease=None,
             # pcolormesh
             kwargs['extend'] = cmap_params['extend']
             kwargs['levels'] = cmap_params['levels']
+            if 'colors' in kwargs:
```

Comment:
Yep, agreed. The way we implemented this, the default levels=None means no levels for color mesh or imshow plots but 7 levels for contour and contourf.

On Thu, Aug 20, 2015 at 9:28 AM, Clark Fitzgerald
<notifications@github.com> wrote:

>> @@ -452,6 +452,8 @@ def newplotfunc(darray, ax=None, xincrease=None, yincrease=None,
>>              # pcolormesh
>>              kwargs['extend'] = cmap_params['extend']
>>              kwargs['levels'] = cmap_params['levels']
>> +            if 'colors' in kwargs:
> @rabernat Good point- we should be able to do this.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/xray/xray/pull/538/files#r37550742